Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -517,25 +560,44 @@ class DropDownMenu
         popUpDiv.style.right = 'auto';
       }
     }
-    if (LayoutUtility.GetHeight (popUpDiv) > space_bottom)
+
+    const popUpHeight = LayoutUtility.GetHeight (popUpDiv);
+    if (popUpHeight > space_bottom - verticalSpacing)
     {
-      if (LayoutUtility.GetHeight (popUpDiv) > document.documentElement.clientHeight)
+      const maximumPopUpHeight = document.documentElement.clientHeight - verticalSpacing * 2;
+      if (popUpHeight > maximumPopUpHeight)
       {
-        popUpDiv.style.top = '0';
+        popUpDiv.classList.add("scrollable");
+
+        // Ensure that the popup retains the minimum size, reducing the spacing if it gets to small;
+        // This prevents the popup from disappearing completely
+        const minimumPopUpHeight = verticalSpacing * 6;
+        const targetPopUpHeight = Math.max(minimumPopUpHeight, maximumPopUpHeight);
+        const remainingSpacing = Math.floor(document.documentElement.clientHeight - verticalSpacing - targetPopUpHeight);
+        const spacing = Math.max(0, remainingSpacing);
+
+        popUpDiv.style.top = spacing + 'px';
```

Comment:
naming: here be basically have two types of vertical spacing. One that's the baseline for the calculation (`verticalSpacing`) and one that's getting applied (`spacing`). We need to find better names for these, probably starting with renaming `verticalSpacing` to give it more semantic